docs/deployment/rds-s3/README.md

```diff
@@ -54,95 +88,129 @@ eksctl create cluster \
-      Ref: SecurityGroupId
-  DeletionPolicy: Snapshot
-
-  ...
-```
+    To complete the following steps you will need to keep track of the following:
+    - `RDS database name` (not to be confused with the `DB identifier`)
+    - `RDS database admin username`
+    - `RDS database admin password`
+    - `RDS database endpoint URL`
+    - `RDS database port`
 
 3. Create Secrets in AWS Secrets Manager
 
-   1. Configure a secret named `rds-secret` with the RDS DB name, RDS endpoint URL, RDS DB port, and RDS DB credentials that were configured when following the steps in Create RDS Instance.
-      - For example, if your database name is `kubeflow`, your endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, your DB port is `3306`, your DB username is `admin`, and your DB password is `Kubefl0w` your secret should look like:
-      - **Note:** These are the default values for the database name and credentials in cloudformation template for creating the RDS instance, change these according to the values you used
-      - ```
-        aws secretsmanager create-secret --name rds-secret --secret-string '{"username":"admin","password":"Kubefl0w","database":"kubeflow","host":"rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com","port":"3306"}' --region $CLUSTER_REGION
-        ```
-   1. Configure a secret named `s3-secret` with your AWS credentials. These need to be long term credentials from an IAM user and not temporary.
-      - Find more details about configuring/getting your AWS credentials here:
-        https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
-      - ```
-        aws secretsmanager create-secret --name s3-secret --secret-string '{"accesskey":"AXXXXXXXXXXXXXXXXXX6","secretkey":"eXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXq"}' --region $CLUSTER_REGION
-        ```
+   1. [RDS] Create the RDS secret and configure the secret provider:
+      1. Configure a secret, for example named `rds-secret`, with the RDS DB name, RDS endpoint URL, RDS DB port, and RDS DB credentials that were configured when following the steps in Create RDS Instance.
+         - For example, if your database name is `kubeflow`, your endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, your DB port is `3306`, your DB username is `admin`, and your DB password is `Kubefl0w` your secret should look like:
+         - ```
+           aws secretsmanager create-secret --name rds-secret --secret-string '{"username":"admin","password":"Kubefl0w","database":"kubeflow","host":"rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com","port":"3306"}' --region $CLUSTER_REGION
+           ```
+      1. Rename the `parameters.objects.objectName` field in [the rds secret provider configuration](../../../awsconfigs/common/aws-secrets-manager/rds/secret-provider.yaml) to the name of the secret. 
+         - Two line command:
+           ```
+           export RDS_SECRET = <your rds secret name>
```

Comment:
cant have space around `=` in bash
```
$ export x = "y"
-bash: export: `=': not a valid identifier
```
also suggest using this variable in the create-secret command above as well, same for s3

Reply:
done